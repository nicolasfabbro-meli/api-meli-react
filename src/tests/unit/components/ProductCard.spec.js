import React from 'react';
import Component from '../../../components/ProductCard';

describe('ProductCard', () => {
  it('Should match snapshot', () => {
    const props = {
      product: {
        permalink: '__LINK__',
        thumbnail: '__IMAGE__',
        title: '__TITLE__',
        currency_id: '__CURRENCY_ID__',
        price: 999,
        shipping: {
          free_shipping: true,
        },
      }
    };
    const component = shallow(<Component {...props} />);
    expect(escapeSnapshot(component)).toMatchSnapshot();
  });
});
