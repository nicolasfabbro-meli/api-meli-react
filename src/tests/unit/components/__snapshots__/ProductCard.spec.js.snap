// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard Should match snapshot 1`] = `
"<a href='__LINK__' className='product-card'>
  <div className='product-card__image'>
    <img src='__IMAGE__' alt='__TITLE__' />
  </div>
  <div className='product-card__info'>
    <span className='product-card__price'>
      __CURRENCY_ID__
       
      999
    </span>
    <span className='product-card__shipping product-card__shipping--hidden'>
      Env√≠o gratis
    </span>
    <span className='product-card__title'>
      __TITLE__
    </span>
  </div>
</a>"
`;
